src/assets/src/components/meetingType.tsx

```diff
@@ -66,8 +66,8 @@ export const BackendSelector: React.FC<BackendSelectorProps> = (props) => {
         props.onChange(event.currentTarget.value);
     }
     return (
-        <select className="btn btn-sm select-dropdown" onChange={handleChange} value={props.selectedBackend}>
+        <Form.Select className="select-dropdown" onChange={handleChange} value={props.selectedBackend}>
             {options}
-        </select>
+        </Form.Select>
```

Comment:
I guess these are syntax change, straightforward, saw application error with starting the app.

Reply:
Did you have a concern here, @pushyamig? Not sure what you're referring to with "saw application error"

Reply:
`saw application error` meaning if you updated the version package.json and did not change it then you might see the app error.

Reply:
Some of these changes are just moving vanilla elements to React Bootstrap components. That's what is happening here.